docs/rules/no-root-v-if.md

```diff
@@ -0,0 +1,37 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/no-root-v-if
+description: disallow `v-if` directives on root element
+---
+
+# vue/no-root-v-if
+
+> disallow `v-if` directives on root element
+
+- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> **_This rule has not been released yet._** </badge>
```

Comment:
To fix the lint warning:

```suggestion
- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> ***This rule has not been released yet.*** </badge>
```

Why did you change the automatically generated (and correct) syntax?

Reply:
Could be a problem with my editor plugin, fixed